client/scss/components/_instagram_thumbnail.scss

```diff
@@ -0,0 +1,51 @@
+.instagram-thumbnail {
+  display: block;
+  position: relative;
+
+  .icon {
+    width: 1em;
+    height: 1em;
+    margin-right: 0.2em;
+  }
+
+  .icon__shape {
+    fill: color('white');
+  }
+}
+
+.instagram-thumbnail__type {
+  position: absolute;
+  top: 2 * $spacing-unit;
+  right: 2 * $spacing-unit;
+  z-index: 1;
+}
+
+.instagram-thumbnail__type-text {
+  @include visually-hidden;
+}
+
+.instagram-thumbnail__overlay {
+  position: absolute;
+  top: 0;
```

Comment:
Aligns the semi-opaque overlay over the image

![screen shot 2017-06-09 at 15 53 29](https://user-images.githubusercontent.com/1394592/26981055-ec80cb2e-4d2b-11e7-83cc-c53bc0d36b10.png)